NEWS.md

```diff
@@ -20,7 +20,7 @@ plumber 1.0.0
 
 ### Breaking changes
 
-* When `plumb()`ing a file (or `plumber$new(file)`), the working directory is set to the file's directory before parsing the file. When running the Plumber API, the working directory will be set to file's directory before running.(#631)
+* When `plumb()`ing a file (or `Plumber$new(file)`), the working directory is set to the file's directory before parsing the file. When running the Plumber API, the working directory will be set to file's directory before running.(#631)
 
```

Comment:
Let's also add a comment about `plumb()` now returning an object of class `Plumber` (not `plumber`), but you can now use `is_plumber()` if you need to check the class.